WTWSMS/history/titles/c_malta.txt

```diff
@@ -9,6 +9,7 @@
 }
 491.1.1 = { #Cassiodorus
 	holder=154006
+	liege = "k_italy"
 }
```

Comment:
Wait, is the 491.1.1 block new? In 428.1.1 Malta is lead by the Vandals.

Reply:
Cassiodorus was added here by @tannerflick4

Reply:
I don't know if Cassiodorus should be here. From what I know Ostrogoths seized Lilybaeum (c_trapani) and probably Malta in 535 during Roman expedition against the Vandals.

Reply:
I am not sure who owned Malta in 491. I probably gave him Malta since it was in the realm he was ruler of as a vassal of Italy. So if @IhateTrains knows more, then its ok to fix this county.

Reply:
By "the realm he was ruler of" you mean Sicily?

Reply:
OK, I will fix c_malta and c_trapani then.

Reply:
Small correction on my post on history: Malta was taken by Romans, not Ostrogoths.

Reply:
Then I say revert both, neither of them should have been changed.

Reply:
Hasn't it been addressed with bf6fe53?

Reply:
Yes.